components/apimgt/org.wso2.carbon.apimgt.rest.api.store.v1/src/main/resources/store-api.yaml

```diff
@@ -845,6 +845,7 @@ paths:
       - $ref: '#/components/parameters/requestedTenant'
       - $ref: '#/components/parameters/limit'
       - $ref: '#/components/parameters/offset'
+      - $ref: '#/components/parameters/commenterInfo'
```

Comment:
shall we rename it as `includeCommenterInfo`

Reply:
https://github.com/wso2/carbon-apimgt/pull/9454/commits/219471b4eaadd05fd00560c3a5c735dedb45de6b